library/network_connections.py

```diff
@@ -1554,23 +1554,34 @@ def _complete_kwargs_loglines(self, rr, connections, idx):
             )
             prefix = prefix + (", '%s'" % (c["name"]))
         for r in rr["log"]:
-            yield (r[2], "[%03d] %s %s: %s" % (r[2], LogLevel.fmt(r[0]), prefix, r[1]))
+            yield (r[2], "[%03d] %s %s: %s" % (r[2], LogLevel.fmt(r[0]), prefix, r[1]), r[0])
```

Comment:
This fails in the CI because the line is too long. Please re-format it with the `black` tool.

Reply:
Done!